README.md

```diff
@@ -1,35 +1,54 @@
-# status: Pre-Alpha
+# Current version: Pre-Alpha
 
-# osu-radio
```

Comment:
Add logo design at the top instead of current version? Current version isnt that important, if a stable build is out it'll show up on the side anyway. Removing the plain text for osu-radio is good idea

Reply:
I can remove current version and just leave it like that, since as you said the icon will show up on the side once a stable build is out anyway